src/index.ts

```diff
@@ -7,6 +7,8 @@ import bodyParser from 'body-parser';
 // import zdRoutes from './restapi/zd_routes';
 import mmRoutes from './restapi/mm_routes';
 
+global.fetch = require('node-fetch');
```

Comment:
I recommend putting this import above any local imports (like `./restapi/mm_routes`), since it is changing the environment that modules in that tree may be using.